packages/core/tests/unit/models/report-test.js

```diff
@@ -11,25 +11,58 @@ import DeliverableItem from 'navi-core/models/deliverable-item';
 let Store;
 
 const ExpectedRequest = {
-    table: 'network',
-    columns: [
-      { type: 'timeDimension', field: 'dateTime', parameters: { grain: 'day' } },
-      { type: 'dimension', field: 'property', parameters: {} },
-      { type: 'metric', field: 'adClicks', parameters: {} },
-      { type: 'metric', field: 'navClicks', parameters: {} }
-    ],
     filters: [
       {
-        type: 'timeDimension',
-        field: 'dateTime',
-        parameters: { grain: 'day' },
         operator: 'bet',
-        values: ['2015-11-09 00:00:00.000', '2015-11-16 00:00:00.000']
+        values: ['2015-11-09 00:00:00.000', '2015-11-16 00:00:00.000'],
+        field: 'dateTime',
```

Comment:
Need some `network.dateTime` in this request

Reply:
Yeah, I actually look into this, but it would require a bunch more changes to the v2 converter. Let's tackle that in another issue.